packages/isl/package.desc

```diff
@@ -1,6 +1,6 @@
 repository='git git://repo.or.cz/isl.git'
 bootstrap='./autogen.sh'
-mirrors='http://isl.gforge.inria.fr'
+mirrors='https://gcc.gnu.org/pub/gcc/infrastructure'
```

Comment:
There's no 0.22 version at gcc.gnu.org, maybe that's why some builds are failing.